src/pe_reports/report_generator.py

```diff
@@ -55,6 +62,19 @@ def load_template():
     return prs
 
 
+def load_customers():
+    """Export PowerPoint report set to output directory."""
+    try:
+        names_obj = {}
+        if os.path.getsize(CUSTOMERS) != 0 and os.path.exists(CUSTOMERS):
+            with open(CUSTOMERS) as customers_file:
+                names_obj = json.load(customers_file)
+            return names_obj
```

Comment:
There is no need to define a variable for `names_obj` since it is only used to return a value.
```suggestion
            with open(CUSTOMERS) as customers_file:
                return json.load(customers_file)
```

Reply:
Don't forget to delete line 68 as well.

Reply:
Line 68 is still there.